indigo_api/serializers.py

```diff
@@ -627,6 +628,15 @@ def get_taxonomies(self, instance):
 
         return taxonomies
 
+    def get_taxonomy_topics(self, instance):
+        taxonomies = []
+        for t in instance.taxonomy_topics.all():
```

Comment:
I think we can just make this a list of slugs

Reply:
Done. :+1: